src/NUnitFramework/framework/Assert.cs

```diff
@@ -343,7 +358,9 @@ private static string GetStackTrace()
         {
             string stackTrace = null;
 
-#if PORTABLE
+#if PORTABLE && !NETSTANDARD1_6
+            // TODO: This isn't actually working! Since we catch it right here,
```

Comment:
Good point about possibly dropping portable. If we do, we may want a .NET Standard 1.3 build to replace it. Not sure yet... We can talk about the pros and cons after i finish the PR I am working on and know more.